tests/Unit/SymfonyHttpClientTest.php

```diff
@@ -3081,6 +3082,48 @@ public function testUpdateContentTopicCompletionThrowsExceptionOnNonSuccessfulUp
         $client->updateContentTopicCompletion($contentTopicCompletionUpdate, 1, 2, 3);
     }
 
+    /**
+     * @throws ApiException
```

Comment:
@jolezg this can be deleted as the client will never throw that exception in this test as the D2L response payload is mocked for the happy path.